docs/user-guide.md

```diff
@@ -0,0 +1,31 @@
+
+In QGIS toolbars click on the STAC APIs Browser icon
+
+![image](images/toolbar.png)
+
+
+## Add a STAC Catalog
+
+**STAC Catalog** is a simple, flexible JSON file of links that provides a structure to organize and browse STAC Items
+
+![image](images/add-connection.png)
+
+## Search and add Item into QGIS
+
```

Comment:
We need to provide details about the whole searching workflow before talking about the search filters.